cloudsync/cs.py

```diff
@@ -260,24 +260,39 @@ def start(self, *, daemon=True, **kwargs):
         self.nmgr.notify(Notification(SourceEnum.SYNC, NotificationType.STARTED, None))
         self.nmgr.start(**kwargs)
 
-
     def stop(self, forever=True, wait=True):
```

Comment:
I'm not sure if we should offer the `wait=False` option here at all, but if we do, I think a big comment about the risks is warranted

Reply:
I didn't want to break the interface for this, and as of the current version, the wait=True/False behavior is the same as before -- in both cases we join the smgr and emgr threads, the wait param affects only the nmgr thread.

Will put all this in a comment, since it isn't obvious.